shapeshift/web
# clean up of RawText to text.components (#8)

## Notice

Before submitting a pull request, please make sure you have answered the following:


## Pull Request Type


## Issue (if applicable)
Closes #8 


## Description

Updated RawText to Text variables stored in messages-en.json. Ideally this will be the one file sent out for translations. 

## Testing


1. Pull branch locally and run `yarn` to install new deps
2. Run `yarn dev` this will launch a local copy on http://localhost:3000/
3. Verify all changes worked and nothing breaks.

## Screenshots (if applicable)
